api/server/router/swarm/cluster_routes.go

```diff
@@ -132,7 +132,7 @@ func (sr *swarmRouter) createService(ctx context.Context, w http.ResponseWriter,
 
 	id, err := sr.backend.CreateService(service, encodedAuth)
 	if err != nil {
-		logrus.Errorf("Error creating service %s: %v", id, err)
+		logrus.Errorf("Error creating service: %v", err)
```

Comment:
Actually there is still something I am concerned about. Since `sr.backend.CreateService` will do a lot of things, such as converting, validating,...
If image is also empty in the api request, maybe it is still not a good choice to print that.